src/components/app/reducers.js

```diff
@@ -0,0 +1,40 @@
+import Cookies from 'cookies-js';
+
+import { SET_COOKIE, REMOVE_COOKIE, RESET_STATE } from './actionTypes';
+import { EXPIRATION } from './constants';
+
+const initialState = {
+    isLoggedIn: false,
```

Comment:
i think this is my fault, but the `isLoggedIn` should be set to `null` initially

Reply:
Alright